server/lobbyconnection.py

```diff
@@ -1074,6 +1082,17 @@ def command_modvault(self, message):
             else:
                 raise ValueError('invalid type argument')
 
+    def command_nts(self, message):
+        out = dict(command="command_nts")
+
+        token = self.nts_client.get_token()
+
+        out['IceServers'] = token['IceServers']
+        out['DateCreated'] = token['DateCreated']
```

Comment:
Well, that would be shit. I used the docs: https://www.twilio.com/docs/api/rest/token

If they rename everything for every language-specific api implementation that's terrible.

I'll check.